certora/confs/AaveV3MigrationBundlerV2.conf

```
@@ -0,0 +1,10 @@
+{
+  "files": [
+    "src/migration/AaveV3MigrationBundlerV2.sol",
+    "certora/harness/Constants.sol"
+  ],
```

Let's specify the solidity compiler version used here.
```suggestion
  ],
  "solc": "solc-0.8.24",
```
This makes it more specific and easier to [handle multiple solidity versions](https://www.notion.so/morpho-labs/Handle-solc-versions-9eb28c0c14d64b4381c874e9d5bb94c5) on one system. It will require to rename the binary in the CI accordingly

---

I had that initially (from eg [Metamorpho](https://github.com/morpho-org/metamorpho/blob/fcf3c41d9c113514c9af0bbf6298e88a1060b220/.github/workflows/certora.yml#L49)) then changed it: the bundlers require an exact solidity version, so it is fully specified. I did not see the advantage of repeating the version number here (until 2 versions are needed). Do you think it's still useful in its current state?

---

Yes I think it's worth it, for convenience mostly. That way I don't have to switch what my `solc` binary points to, and can work on multiple verification at the same time

---

02d514d5a699ecec35037e28fbc840114cea3ec5